src/elements/x-safe.js

```diff
@@ -160,6 +160,11 @@ export default class XSafe extends MixinRedux(XElement) {
     }
 
     _onPropertiesChanged(changes) {
+        if (changes.activeWallet === null) {
```

Comment:
Does `changes` contain the actual changes, or just `true` for changed properties? I think it would be safer to check `this.properties.activeWallet` for `null`.

Reply:
It *does* contain the actual property.